packages/muelu/research/mmayr/composite_to_regions/src/SetupRegionHierarchy_def.hpp

```diff
@@ -1326,6 +1383,48 @@ void createRegionHierarchy(const int maxRegPerProc,
 } // createRegionHierarchy
 
 
+// Wrapper to be used from the Matlab-based driver
```

Comment:
That is not a bad idea, as you realize, I was using default values for some parameters to comply with the matlab driver but this looks better.

Reply:
I had trouble to add another argument with default value. It always becomes difficult if there are multiple default values...